people_recognition_3d/src/people_recognition_3d/people_recognizer_3d.py

```diff
@@ -257,7 +257,9 @@ def recognize(self, rgb, depth, camera_info):
                       (rospy.Time.now() - t).to_sec())
         rospy.loginfo('Found {} people'.format(len(people2d)))
 
-        cmap = color_map(N=len(people2d), normalized=True)
+        max_group = max(person2d.body_parts[0].group_id for person2d in people2d) if people2d else 0
```

Comment:
Unclear what groups are.

Reply:
I have added a comment about what groups are. These are the IDs assigned to recognized people by openpose node. 2D recognition node might not select all the people recognized by openpose node so `len(people2d)  =? max_group`.

Ideally these ids should be reassigned to selected recognitions but that would involve quite some refactor.